.gitignore

```diff
@@ -45,6 +45,7 @@ perf.data
 perf.data.old
 
 cpp/.idea/
+.clangd/
```

Comment:
For my enlightenment, what is ".clangd"?

Reply:
`.clangd/` is a cache directory used by the language server `clangd`. It powers autocompletion and other code awareness features